### ISSUE TITLE
invalid pool request pool overlaps with other one on this address space

### ISSUE THREAD

**Issue by ketronix-dev** [Dokploy/dokploy#887]
### To Reproduce

-

### Current vs. Expected behavior

`invalid pool request pool overlaps with other one on this address space` while i install dokploy on vps

### Provide environment information

```bash
_,met$$$$$gg.           
    ,g$$$$$$$$$$$$$$$P.       ------------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 12 (bookworm) x86_64 
 ,$$P'              `$$$.     Host: OpenStack Nova 27.5.2 
',$$P       ,ggs.     `$$b:   Kernel: 6.1.0-28-amd64 
`d$$'     ,$P"'   .    $$$    Uptime: 55 mins 
 $$P      d$'     ,    $$P    Packages: 501 (dpkg) 
 $$:      $$.   -    ,d$$'    Shell: bash 5.2.15 
 $$;      Y$b._   _,d$P'      Resolution: 1280x800 
 Y$$.    `.`"Y$$$$P"'         Terminal: /dev/pts/0 
 `$$b      "-.__              CPU: Intel Xeon (Skylake, IBRS) (4) @ 2.394GHz 
  `Y$$                        GPU: 00:02.0 Red Hat, Inc. Virtio 1.0 GPU 
   `Y$$.                      Memory: 3802MiB / 7940MiB 
     `$$b.
       `Y$$b.                                         
          `"Y$b._                                     
              `"""
```

### Which area(s) are affected? (Select all that apply)

Installation

### Are you deploying the applications where Dokploy is installed or on a remote server?

Remote server

### Will you send a PR to fix it?

No

**Comment by Siumauricio** [comment]
I don't think this is a bug with dokploy itself, but rather a problem with your operating system.

You can check it by directly googling the message
https://stackoverflow.com/questions/56515128/error-pool-overlaps-with-other-one-on-this-address-space-when-starting-my-proje

**Comment by kavinask007** [comment]
For anyone facing this issue in the future . this happens when the dokploy-network subnet overlaps with exisiting networks subnet
you can check this by running `docker network inspect $(docker network ls -q) --format '{{.Name}}: {{range .IPAM.Config}}{{.Subnet}}{{end}}'`
In my case it overlapped with bridge network `bridge: 10.0.1.0/24
docker_gwbridge: 172.17.0.0/16
dokploy-network: 10.0.1.0/24
host:
ingress: 10.0.0.0/24
none:`

Fix :  Remove the service and network . follow the manual installation from https://docs.dokploy.com/docs/core/manual-installation
but in the network creating stage use a diff subnet 
ex `docker network create --subnet=10.0.2.0/24 --driver overlay --attachable dokploy-network`

**Comment by gonzague** [comment]
I'm having the same issue when installing Dokploy on a host where Docker has been used before (but is empty of containers etc) . This is a Debian 12 host.

```
root@server:~/backend# curl -sSL https://dokploy.com/install.sh | sh
Docker already installed
Node left the swarm.
Using advertise address: 10.0.0.4
Swarm initialized: current node (l8tqx2zo9fx1zhdn871kdq36j) is now a manager.

To add a worker to this swarm, run the following command:

    docker swarm join --token SWMTKN-1-xxxxx-89o8pebsp9tp0ioub6mp4tlby 10.0.0.4:2377

To add a manager to this swarm, run 'docker swarm join-token manager' and follow the instructions.
Swarm initialized
l3p8njm8avugj2iodqfz2ogxl
Network created
5qj7j0evf66fupfx2pr3n2t6d
overall progress: 0 out of 1 tasks 
1/1: invalid pool request: Pool overlaps with other one on this address space
```

The bizarre thing is that this host only has the "default" Docker networks:
```
root@server:~/backend# docker network ls
NETWORK ID     NAME      DRIVER    SCOPE
d8226e5a386b   bridge    bridge    local
6163b05ae544   host      host      local
e5ba42ab2336   none      null      local
```

So I cant delete any of these :)

**Comment by gonzague** [comment]
After that I completely uninstalled Docker from the machine ( running `sudo apt-get purge -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin` + ```
rm -rf /var/lib/docker
rm -rf /var/lib/containerd
```

Restarted the install script and got the same behavior : 

```
root@server-api:~/backend# curl -sSL https://dokploy.com/install.sh | sh
# Executing docker install script, commit: bedc5d6b3e782a5e50d3d2a870f5e1f1b5a38d5c
+ sh -c apt-get -qq update >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get -y -qq install ca-certificates curl >/dev/null
+ sh -c install -m 0755 -d /etc/apt/keyrings
+ sh -c curl -fsSL "https://download.docker.com/linux/debian/gpg" -o /etc/apt/keyrings/docker.asc
+ sh -c chmod a+r /etc/apt/keyrings/docker.asc
+ sh -c echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian bookworm stable" > /etc/apt/sources.list.d/docker.list
+ sh -c apt-get -qq update >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get -y -qq install docker-ce docker-ce-cli containerd.io docker-compose-plugin docker-ce-rootless-extras docker-buildx-plugin docker-model-plugin >/dev/null
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_CTYPE = "UTF-8",
	LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_US.UTF-8").
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
+ sh -c docker version
Client: Docker Engine - Community
 Version:           28.3.3
 API version:       1.51
 Go version:        go1.24.5
 Git commit:        980b856
 Built:             Fri Jul 25 11:34:00 2025
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          28.3.3
  API version:      1.51 (minimum version 1.24)
  Go version:       go1.24.5
  Git commit:       bea959c
  Built:            Fri Jul 25 11:34:00 2025
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.7.27
  GitCommit:        05044ec0a9a75232cad458027ca83437aae3f4da
 runc:
  Version:          1.2.5
  GitCommit:        v1.2.5-0-g59923ef
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

================================================================================

To run Docker as a non-privileged user, consider setting up the
Docker daemon in rootless mode for your user:

    dockerd-rootless-setuptool.sh install

Visit https://docs.docker.com/go/rootless/ to learn about rootless mode.


To run the Docker daemon as a fully privileged service, but granting non-root
users access, refer to https://docs.docker.com/go/daemon-access/

WARNING: Access to the remote API on a privileged Docker daemon is equivalent
         to root access on the host. Refer to the 'Docker daemon attack surface'
         documentation for details: https://docs.docker.com/go/attack-surface/

================================================================================

Using advertise address: 10.0.0.4
Swarm initialized: current node (tqxt42n8kydnqqh9kz3tjroq6) is now a manager.

To add a worker to this swarm, run the following command:

    docker swarm join --token SWMTKN-1-xxxx-8owr9qafyd63c5pla75bppspv 10.0.0.4:2377

To add a manager to this swarm, run 'docker swarm join-token manager' and follow the instructions.
Swarm initialized
o0bobmx8h4s6g6kv4xg0vjwis
Network created
y1i97rmqhozfjma3a5daps5m9
overall progress: 0 out of 1 tasks 
1/1: invalid pool request: Pool overlaps with other one on this address space 
```

**Comment by gonzague** [comment]
Last update **found the problem** : Hetzner private networking was using adresses in the 10.0.0.X space :)

**Comment by melon-husk** [comment]
I spent considerable time fixing this issue, on Oracle VM, 
`bridge => 10.0.1.0/24` is reserved

so in the script, you have to update `docker network create` ln to the following, you can put any subnet which is available,

<img width="1263" height="145" alt="Image" src="https://github.com/user-attachments/assets/407feade-17cf-48a0-8fe1-79deee1fbaea" />

this fixed it for me.

**Comment by m-primo** [comment]
Not sure why this was closed and the `bug` tag was removed, it's definitely a bug, thanks to @kavinask007 comment, it worked for me.

And to add more, to remove the service:
`docker service ls`
then get the id and remove that service:
`docker service rm <id>`
After that, remove the network:
`docker network rm dokploy-network`
then in the manual installation script search for: `docker network create`, you'll find a line like this:
`docker network create --driver overlay --attachable dokploy-network`
replace it to:
`docker network create --subnet=10.0.2.0/24 --driver overlay --attachable dokploy-network`

This is exactly what worked for me.

**Comment by felipegcoutinho** [comment]
Thank you!!! @m-primo and @kavinask007
